content/slice/language-guide/exception.md

```diff
@@ -117,66 +127,41 @@ exception DerivedException : BaseException {
 ```csharp
 public partial class BaseException : SliceException
 {
-    public int ErrorCode;
+    public int ErrorCode { get; set; }
 
-    // Primary constructor
-    public BaseException(
-        int errorCode,
-        string? message = null,
-        System.Exception? innerException = null)
-        : base(message, innerException)
+    public BaseException()
+    {
+    }
+
+    public BaseException(int errorCode)
     {
-        ...
+        this.ErrorCode = errorCode;
     }
 }
 
 public partial class DerivedException : BaseException
 {
-    public double Measurement;
+    public double Measurement { get; set; }
+
+    public DerivedException()
+    {
+    }
 
-    // Primary constructor.
     public DerivedException(
         int errorCode,
-        double measurement,
-        string? message = null,
-        System.Exception? innerException = null)
-        : base(errorCode, message, innerException)
+        double measurement)
+        : base(errorCode)
     {
-        ...
+        this.Measurement = measurement;
     }
 }
 ```
 
 {% /aside %}
 
-### ConvertToInternalError
```

Comment:
This is bogus. We only support ConvertToInternalError for DispatchException, and they are unrelated to Slice exceptions:

https://github.com/icerpc/icerpc-csharp/blob/b1be5453abe350afaff5338fc3e2b7eaf151bab4/src/IceRpc/DispatchException.cs#L16